src/components/Tree/Item/index.js

```diff
@@ -142,6 +29,16 @@ const TreeItem = ({
   style,
   setOpen,
 }) => {
+  const tooltipClasses = useTooltipStyles()
+  const ref = useRef()
```

Comment:
extract as a hook? `EllipsisBox` has the same code

Reply:
they're slightly different but I think we can refactor it

Reply:
Yeah, it can be enhanced of course, could you help?